ietf/utils/db.py

```diff
@@ -0,0 +1,27 @@
+# Copyright The IETF Trust 2021, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+# Taken from/inspired by
+# https://stackoverflow.com/questions/55147169/django-admin-jsonfield-default-empty-dict-wont-save-in-admin
+# 
+# JSONField should recognize {}, (), and [] as valid, non-empty JSON
+# values.  However, the base Field class excludes them
+import jsonfield
+from django.forms import fields
+
+from ietf.utils.fields import IETFJSONField
```

Comment:
It'd be a good idea to avoid degenerate class names in this file. If you just `import ietf.utils.fields`, the form_class would then be `ietf.utils.fields.IETFJSONField`. You could also do `from ietf.utils.fields import IETFJSONField as FormIETFJSONField` (or similar).

Reply:
I feel your pain! (and I accept your patch!)